README.rst

```diff
@@ -148,6 +148,8 @@ You can use this `Dockerfile`_ to launch/execute the Docker image and container:
     $ docker build -t fiwareaiakos .
     $ docker run -p 3000:3000 -d fiwareaiakos
 
+If you want to get more information about the use of docker see `How to use Aiakos with Docker <docker/README.md>`_.
+
```

Comment:
I think previous documentation is out of date. Now the fiware aiakos image is called fiware-aiakos right?